src/Analyzers/Core/CodeFixes/AddParameter/AbstractAddParameterCodeFixProvider.cs

```diff
@@ -244,7 +245,7 @@ private static int NonParamsParameterCount(IMethodSymbol method)
 
         ImmutableArray<CodeAction> NestByOverload()
         {
-            using var _ = ArrayBuilder<CodeAction>.GetInstance(codeFixData.Length, out var builder);
+            var builder = new FixedSizeArrayBuilder<CodeAction>(codeFixData.Length);
             foreach (var data in codeFixData)
```

Comment:
note: all loops were audited to make sure they always add an item for each input iterated over.  loops that did not do this were not updated.